### PR TITLE
feat: #MED-207 add devController for easy testing

### PR DESCRIPTION
**Pull request by BastienBenard**
## Describe your changes
Added dev API to simulate a GAR response
Added reflectionsVersion=0.10.2 dependencies for testing

## Checklist tests
None

## Checklist before requesting a review (magic string, indentation, comment/documentation...)

- [x] I have detailed the tests to do in my feature/fix in order to prevent consequents regressions (must specify in **Checklist tests**)
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (API Doc etc...) - (must specify in **Description** for target version)
- [ ] If it is a consequent feature, I have added thorough tests.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been added to this project (must specify in **Description**)

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -70,6 +70,7 @@ dependencies {
   testCompile "org.powermock:powermock-api-mockito2:$powerMockVersion"
   testCompile "org.powermock:powermock-module-junit4:$powerMockVersion"
   testCompile "org.powermock:powermock-module-testng:$powerMockVersion"
+  testCompile "org.reflections:reflections:$reflectionsVersion"
```
Comment (BastienBenard) [inline build.gradle:73]: Permet de faire des tests

File: src/test/java/fr/openent/gar/helper/StringHelperTest.java
Comment (BastienBenard) [inline src/test/java/fr/openent/gar/helper/StringHelperTest.java:1]: Repris de CRRE

File: src/test/java/fr/openent/gar/helper/IModelHelperTest.java
Comment (BastienBenard) [inline src/test/java/fr/openent/gar/helper/IModelHelperTest.java:1]: Repris de CRRE

File: src/main/java/fr/openent/gar/model/IModel.java
Comment (BastienBenard) [inline src/main/java/fr/openent/gar/model/IModel.java:1]: Repris de CRRE

File: src/main/java/fr/openent/gar/helper/IModelHelper.java
Comment (BastienBenard) [inline src/main/java/fr/openent/gar/helper/IModelHelper.java:1]: Repris de CRRE

File: src/main/java/fr/openent/gar/helper/StringHelper.java
Comment (BastienBenard) [inline src/main/java/fr/openent/gar/helper/StringHelper.java:1]: Repris de CRRE

**Review by JinDouang — APPROVED** [review]

**Review by flmartineau — APPROVED** [review]

**Pull request merged**